libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -61,6 +61,21 @@ export class StripeManager {
     });
   }
 
+  /**
+   * Cancels incomplete subscription
+   */
+  async cancelIncompleteSubscription(customerId: string, priceId: string) {
+    const subscriptions = await this.getSubscriptions(customerId);
+    const targetSub = subscriptions.data.find(
+      (sub) => sub.items.data.find((item) => item.price.id === priceId) !== null
+    );
```

Comment:
Do we need to consider what if there are multiple subscriptions? Should this method be renamed to `cancelIncompleteSubscriptionsToPrice`?